pipeline_dp/partition_selection.py

```diff
@@ -20,7 +20,6 @@ def create_partition_selection_strategy(
         strategy: pipeline_dp.PartitionSelectionStrategy, epsilon: float,
         delta: float,
         max_partitions_contributed: int) -> "PartitionSelectionStrategy":
-
     if strategy == pipeline_dp.PartitionSelectionStrategy.TRUNCATED_GEOMETRIC:
```

Comment:
Please return back

Reply:
We return at the end of the function, do you mean undo deleting the newline?

Reply:
yes